src/Symfony/Bundle/FrameworkBundle/Resources/config/collectors.xml

```diff
@@ -53,5 +53,10 @@
             <tag name="kernel.event_listener" event="kernel.controller" method="onKernelController"/>
             <tag name="data_collector" template="@WebProfiler/Collector/router.html.twig" id="router" priority="285" />
         </service>
+
+        <service id="data_collector.http_client" class="Symfony\Bundle\FrameworkBundle\DataCollector\HttpClientDataCollector">
+            <argument type="service" id="Symfony\Contracts\HttpClient\HttpClientInterface" on-invalid="ignore" />
+            <tag name="data_collector" template="@Framework/Collector/http_client.html.twig" id="http_client" priority="240" />
```

Comment:
the template should be in WebProfilerBundle for consistency with other core panels.